### PR TITLE
Scaffold app

### PR DESCRIPTION
**Pull request by vgeorge**
At the moment it just does linting and testing.

### CODE REVIEW

**Comment by vgeorge** [comment]
I've created this to test CircleCI, but I'll use to build the scaffold too. Changed title fo WIP.

**Comment by vgeorge** [comment]
@LanesGood, I've added a React app structure based on recent projects, with routes `/traces`, `/photos`, `/about` and `/sandbox`. It includes collecticons, button and form components. I can remove if you think it is not the case to have them now or we can clean up as we go.

The feedback I need here is to check if the README instructions are ok and you can build the app successfully.

I've discussed with @ricardoduplos if it is the case of setting up a "Design System" repo. I'm not sure if the mobile app can benefit from that, but it might be a good for the long-term. If you think that is the case I can help with the setup.

File: README.md [hunk 1]
```diff
@@ -1,91 +1,54 @@
-# project-seed
+# Observe Dashboard
 
-A basic starting point for web projects
+A dashboard to browse and manage Observe platform data.
```
Comment (LanesGood) [inline README.md]:
```suggestion
A dashboard to browse and manage data from the [Observe](https://github.com/developmentseed/observe) mobile mapping application.
```

**Comment by vgeorge** [comment]
Thanks, @LanesGood.

**Pull request merged**